main.js

```diff
@@ -1,32 +1,41 @@
+let url = window.location.toString();
 
-
-let str = "https://github.com/?username=6thSence";
 let nickName = (link) => {
   let arr = link.split('=');
-  return (arr[arr.length-1]);
+  let nick = arr[1];
+  if(nick === undefined) {
+    nick = "x1nax";
+  }
+  return nick;
 }
-nickName(str);
 
 let info = (user) => {
   fetch('https://api.github.com/users/'+ user)
-    .then(res => res.json())
-      .then(res => {
-        let newImg = document.createElement("IMG");
-        let newh1 = document.createElement("h1");
-        let newp = document.createElement("p");
-        let newA = document.createElement("a");
-        newImg.src = res.avatar_url;
-        newh1.innerHTML = res.name;
-        newp.innerHTML = res.bio;
-        newA.href = res.html_url;
-        document.body.appendChild(newA);
-        newA.appendChild(newh1);
-        document.body.appendChild(newp);
-        document.body.appendChild(newImg);
-
-        }
-      );
-
+  .then(res => {
+    if (res.status !== 404) {
+      return res.json();
+    }
+    else {
+      let err = new Error;
+      err.res = res;
+      throw err;
+      }
+  })
+  .then(res => {
+    let newImg = document.createElement("IMG");
+    let newp = document.createElement("p");
+    let newA = document.createElement("a");
+    let newh1 = document.createElement("h1");
+    newImg.src = res.avatar_url;
+    newh1.innerHTML = res.name;
+    newp.innerHTML = res.bio;
+    newA.href = res.html_url;
+    document.body.appendChild(newA);
+    newA.appendChild(newh1);
+    document.body.appendChild(newp);
+    document.body.appendChild(newImg);
+    })
+    .catch(err => document.body.innerHTML = "Такой пользователь не существует. Ошибка: "+err);
```

Comment:
Codacy found an issue: [Unsafe assignment to innerHTML](https://app.codacy.com/gh/WebHeroSchool/jstask35/pullRequest?prid=6095927)